diff --git a/node_modules/sass/sass.dart.js b/node_modules/sass/sass.dart.js
index 403b3df..1d9757b 100644
--- a/node_modules/sass/sass.dart.js
+++ b/node_modules/sass/sass.dart.js
@@ -119719,7 +119719,7 @@ self.util = _cliPkgRequires.util;
         _this.whitespace$0();
         argument = _this.importArgument$0();
         t3 = argument instanceof A.DynamicImport0;
-        if (t3)
+        if (t3 && false)
           t2.push(new A._Record_3_deprecation_message_span(B.Deprecation_A0i, string$.Sassx20_i, argument.span));
         if ((_this._stylesheet0$_inControlDirective || _this._stylesheet0$_inMixin) && t3)
           _this._stylesheet0$_disallowedAtRule$1(start);
